app/src/api/jobs.js

```diff
@@ -28,6 +28,21 @@ export const getAllJobsAndSkills = async () => {
   }
 };
 
+export const createJob = async (jobName, jobDesc) => {
```

Comment:
Maybe cna rename to createJobWithJobNameAndDesc

Reply:
I think don't need to be so verbose for this? cos the only way to create a job is with name and desc